### ISSUE TITLE
Feature Request Add .env to php

### ISSUE THREAD

**Issue by mfr75** [nanoninja/docker-nginx-php-mysql#37]
Would be great to have access to .env values in PHP to avoid having to connect to the database and reuse db_name, db_user etc etc

**Comment by nanoninja** [comment]
You are right. I have to think about doing a version 2. Because there are lots of ideas and basically I didn't think that this repository would be useful to many people.
